translations/messages+intl-icu.de.yml

```diff
@@ -2707,8 +2707,11 @@ procedure.phases.external.evaluating: "Auswertung Öffentlichkeit"
 procedure.phases.external.earlyparticipation: "Frühzeitige Beteiligung Öffentlichkeit"
 procedure.phases.external.participation: "Öffentliche Auslegung"
 procedure.phases.external.closed: "Beschlussfassung Öffentlichkeit"
+procedure.pictogram: "Übersichtskarte/Bild"
+procedure.pictogram.altText: "Alternativtext des Bildes"
+procedure.pictogram.altText.toolTipp: "Geben Sie eine kurze Beschreibung des Bildinhalts ein, um die Barrierefreiheit zu gewährleisten."
+procedure.pictogram.copyright: "Copyright des Bildes"
```

Comment:
Do we say 'Copyright' or 'Urheberrecht'?

Reply:
I haven't thought about this before but mixing german and english words is kind of odd, however I couldn't find in slack or in the ticket anything in that regard hence I trust my gut here (and that you find it strange too) and change it to `Urheberrecht`. https://github.com/demos-europe/demosplan-core/pull/3999/commits/dc3a96c8068ac567a51ebc216b65b19c497ba8fa